galaxy.yml

```diff
@@ -48,18 +83,10 @@ build_ignore:
   - tests 
   - .release-it.yml 
   - .travis.yml 
-  - ansible.cfg.saved 
   - CHANGELOG.md 
-  - no-ansible.cfg
-  - run-delete-ims-datasets
-  - run-deprov-wazi 
-  - run-prov-wazi 
-  - update-operator.sh 
   - build
   - Jenkinsfile
   - ansible-lint-config.yml
   - Dockerfile 
-  - playbooks/test.yml
-  - playbooks/delete-ims-datasets.yml
```

Comment:
Guessing these files are no longer relevant or needed to be built as part of the collection.

Reply:
yes